### DISCUSSION TITLE
Next.js 13 - appDir - Can't get the router with useRouter

### DISCUSSION THREAD

**Discussion by NicolasMarafetti** [vercel/vercel#8892] [Help] [answered]
Hello,

I try to get the router with the useRouter hook.
I'm using Next.js 13 and the new app folder structure.

When I go to the page which use useRouter() I have the error 
NextRouter was not mounted.

You can find a minimal github to reproduce the error here : https://github.com/NicolasMarafetti/next-13-router-not-mounted-bug

If you want to see my page which cause the error I give you the code below :

```
'use client';

import { useRouter } from 'next/router';

export default function page() {

  const router = useRouter();

  return (
    <div>page</div>
  )
}
```

My local Next.js version: 13.0.1
My local node version: 16.16.0

I don't know if I'm doing something wrong or if this is a next.js 13 bug (which I completely understand it's beta)
Thanks for the support

**Comment by NicolasMarafetti** [comment]
EDIT

I removed Typescript and Tailwind from the github repo to simplify.
The problem still persist.

**Comment by NicolasMarafetti** [comment]
I tried to use the same files in a /pages directory and all worked fine, I could access the router with useRouter().

I added the folder /pages on github if you want to test it.

So I presume this is an appDir error.

**Comment by amyegan** [comment, marked as answer]
Hi @NicolasMarafetti! I think you need to change the [`useRouter`](https://beta.nextjs.org/docs/api-reference/use-router) import from "next/router" to "next/navigation" like `import { useRouter } from 'next/navigation';`. The example repo worked for me when I modified that import.

**Comment by NicolasMarafetti** [comment]
Oh yes that's true, I didn't understand clearly the difference between next/router and next/navigation, I thought it was like one for the client components one for server components but no.

To resume Next.js anterior to v13 use ```next/router```.
Next.js v13 and later use ```next/navigation```.

Thank you @amyegan for the help
